x/ethbridge/client/cli/tx.go

```diff
@@ -417,3 +418,35 @@ func GetCmdSetBlacklist() *cobra.Command {
 
 	return cmd
 }
+
+func GetCmdPauser() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "set-pauser [pause]",
```

Comment:
Why is this called set-pauser? This appears to pause or unpause not add a pauser roll.